packages/schemer/src/index.ts

```diff
@@ -3,6 +3,7 @@ import addFormats from 'ajv-formats';
 import fs from 'fs';
 import traverse from 'json-schema-traverse';
 import get from 'lodash/get';
+import lowerFirst from 'lodash/lowerFirst';
```

Comment:
the unwritten rule is to never introduce any extra lodash usage. Maybe this works:

```ts
function lowerFirst(str: string): string {
    return str.charAt(0).toLowerCase() + str.slice(1);
}
```